applications/GeoMechanicsApplication/python_scripts/geomechanics_U_Pw_solver.py

```diff
@@ -116,6 +116,9 @@ def AddDofs(self):
             node.AddDof(KratosGeo.TOTAL_DISPLACEMENT_X)
             node.AddDof(KratosGeo.TOTAL_DISPLACEMENT_Y)
             node.AddDof(KratosGeo.TOTAL_DISPLACEMENT_Z)
+            node.AddDof(KratosGeo.INCREMENTAL_DISPLACEMENT_X)
+            node.AddDof(KratosGeo.INCREMENTAL_DISPLACEMENT_Y)
+            node.AddDof(KratosGeo.INCREMENTAL_DISPLACEMENT_Z)
```

Comment:
I'm still not sure what the difference is between dof and solution step variable. Does it crash if these lines are not added?

Reply:
Adding/removing has no effect ( nor for total displacement, velocities and accelerations )
Removed my addition, the others can be removed in a separate PR.